docs/operations/hotplug_interfaces.md

```diff
@@ -120,6 +127,27 @@ See the [Live Migration](./live_migration.md) docs for more details.
 
 Once the migration is completed the VM will have the new interface attached.
 
+### Remove interface
+```bash
+virtctl removeinterface vm-fedora --name dyniface1
+```
+At this point the new interface should be detached from the guest but exist in the pod.
+
+### Migrate the VM
```

Comment:
Please mention the mogration should be done only after the VMI spec has the interface marked as absent (if you can, please add similar clarification to the hotplug as well.)

Reply:
We need to be careful on how this is phrased here.
I suggest to add a note, recommending to avoid performing migrations in parallel to a hotplug. It is safer to assure the hotplug succeeded or at least reached the VMI specification before issuing a migration.

Reply:
Or added a note.

Reply:
Yes, I commented on it. Too low level IMO, regular users will not understand it for sure.